src/color-library.ts

```diff
@@ -57,10 +107,20 @@ export function getColorBrightness(input = '') {
   return tinycolor(input).getBrightness();
 }
 
+export function getReadabilityRatio(
+  backgroundColor = '',
+  foregroundColor = ''
+) {
+  return tinycolor.readability(
+    tinycolor(backgroundColor),
+    tinycolor(foregroundColor)
+  );
+}
+
 export function isValidColorInput(input: string) {
   return tinycolor(input).isValid();
 }
 
-function formatHex(color: tinycolorInstance) {
+function formatHex(color: tinycolor.Instance) {
```

Comment:
Is this correct? Did you intend to use the namespace/type?

Reply:
Yes, this is a change from tinycolor to tinycolor2.